### DISCUSSION TITLE
How to remove / detach a relation?

### DISCUSSION THREAD

**Discussion by rbartholomay** [CodeDredd/pinia-orm#1379] [Q&A]
How can a remove a relation?

I have set up two models with the  @OnDelete('cascade' decorator. All runs fine. When a delete the model, all related childs are also deleted.

But how can a remove / detach a specific child from the relation?

Bye, René

**Comment by oaubert** [comment]
Facing the same issue, the direction I am taking is implementing a generic clone method for objects. Then, you don't really remove a child from the relation, but clone it (without its link to the parent), delete it then link the clone to the appropriate parent.

**Comment by CodeDredd** [comment]
That sounds like a new feature. @oaubert is right. This is the best way to remove it. I think a new method on the model should resolve the problem if i understood it correctly.
